### ISSUE TITLE
How To configure a Basic Authentication?

### ISSUE THREAD

**Issue by pinalbhatt** [jensoleg/swagger-ui#71]
I need to pass basic auth header to my api. Just like Basic swagger UI do we have option to add username / password to create basic auth header?

**Comment by german-bortoli** [comment]
Indeed, I am researching how to add the flow password on this, did not find a cool solution
